extension-requests/style.css

```diff
@@ -40,7 +40,7 @@ body {
 .container {
   width: 100%;
   text-align: center;
-  padding: 10px;
+  padding: 40px;
 }
```

Comment:
please use rem instead of px, use px only where the size is fixed for all screen-sizes.

Reply:
done